Kubernetes/flannel/l2bridge/net-conf.json

```diff
@@ -1,5 +1,5 @@
 {
-  "Network": "192.168.0.0/16",
+  "Network": "10.244.0.0/16",
```

Comment:
Make sure users know to specify their own Cluster CIDR here. It might be different than 10.244/16

Reply:
Yes, I have this included in the [instructions](https://github.com/Microsoft/SDN/blob/dce7ecce37c5e4a876b172bb6cfc126ef22c5a01/Kubernetes/flannel/l2bridge/ReadMe.md#3-download-the-contents-of-l2bridge-directory-into-ck-and-do-the-following)